dcos/cosmos.py

```diff
@@ -45,8 +45,8 @@ def _data(versions, http_method):
         self._endpoint_data = {
             'capabilities': _data(['v1'], 'get'),
             'package/add': _data(['v1'], 'post'),
-            'package/describe': _data(['v2', 'v1'], 'post'),
-            'package/install': _data(['v2', 'v1'], 'post'),
```

Comment:
@klueska After this change this version of the CLI will only work against 1.10 clusters.